std/algorithm/iteration.d

```diff
@@ -788,24 +801,6 @@ Params:
     pred = predicate to apply to each element of the range
     r = range or iterable over which each iterates
 
-Example:
----
-void deleteOldBackups()
-{
-    import std.algorithm, std.datetime, std.file;
-    auto cutoff = Clock.currTime() - 7.days;
-    dirEntries("", "*~", SpanMode.depth)
-        .filter!(de => de.timeLastModified < cutoff)
-        .each!remove();
-}
----
-
-If the range supports it, the value can be mutated in place. Examples:
----
-arr.each!((ref a) => a++);
-arr.each!"a++";
----
```

Comment:
This example probably should be reflected in a ddoc'd unittest too. There's one in the current unittest but it's rather obscure; perhaps this particular example should be expanded to be compilable, and put in its own unittest block. Don't forget to include the sentence above it, as it is explaining an important feature of this function that the user should know, so it should either be in the ddoc comment before the unittest block, or in a comment in the unittest itself.